### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by GSK-FS** [skiptools/skip#28]
I have latest xcode 15, java jdk 8 installed.

[✓] Skip version 0.7.14 (= 0.7.14)
[✓] macOS version 14.1 (> 13.5.0)
[✓] Swift version 5.9 (= 5.9.0)
[✗] Xcode version: error executing xcodebuild
[✓] Homebrew version 4.1.17 (> 4.1.0)
[✓] Gradle version 8.4 (> 8.3.0)
[✗] Java version could not extract version from java
[✓] Android Debug Bridge version 1.0.41 (> 1.0.40)
[✓] Android Studio version: 2022.3
[✗] Create project hello-skip (0.06s)

**Comment by andresantana-dev** [comment]
Check issue #18 , if I'm not wrong someone added a solution

**Comment by marcprux** [comment]
Yes, https://github.com/skiptools/skip/issues/18 addresses it. Please post follow-ups there.
